tests/mongodb/unit/test_crud_operations.py

```diff
@@ -52,19 +85,40 @@ def test_find_documents_with_projections(self, mongodb_store ):
         assert "age" in results[0]
         assert "email" not in results[0]
 
-    def test_find_documents_with_skip_limit(self, mongodb_store ):
+    def test_find_documents_with_skip_limit(self, mongodb_store):
         """Test finding documents with skip and limit parameters."""
         for i in range(5):
             doc = {"name": f"User {i}", "age": 20 + i}
             mongodb_store.insert(TEST_COLLECTION, doc)
         results = mongodb_store.find(TEST_COLLECTION, skip=2, limit=2)
         assert len(results) == 2
 
+    def test_find_documents_with_ordering(self, mongodb_store):
```

Comment:
I am okay with this happy test case. Can you add some more cases like using: 
- 1 order
- empty orders
- attribute which is not appeared in the collection
@Kwang1402